### PR TITLE
update icons

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request updates the matched-zoom icons to be more self-consistent.

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by pllim** [comment]
I guess it is okay though I can barely make out the "chain" on the icon. It is very small on my screen.

Screenshot 2023-09-14 120741

**Comment by kecnry** [comment]
note: this should be squashed and merged to squash out the intermediate commit

File: jdaviz/data/icons/zoom_xrange.svg
Comment (pllim) [inline jdaviz/data/icons/zoom_xrange.svg:1]: I see no difference, so why is it reportedly changed?

Comment (kecnry) [reply jdaviz/data/icons/zoom_xrange.svg:1]: its horizontally centered now

Comment (pllim) [reply jdaviz/data/icons/zoom_xrange.svg:1]: Ahhhh I see it now. Thanks for the clarification!

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**